pkg/reactable/reactable.go

```diff
@@ -293,6 +293,9 @@ func (r *Reactable) AuraContains(e ...attributes.Element) bool {
 		if v == attributes.Cryo && r.Durability[Frozen] > ZeroDur {
 			return true
 		}
+		if v == attributes.Burning && r.Durability[BurningFuel] > ZeroDur && r.Durability[Burning] > ZeroDur {
+			return true
+		}
```

Comment:
Instead of having this check as part of AuraContains, more sensible to just add a new helper function call it `IsBurning() bool` or something and do the same check. This way you don't need the new element as well